includes/omise-wc-setting.php

```diff
@@ -0,0 +1,100 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+return array(
```

Comment:
@nimid for the reason that I asked you to complete the features within this PR (without waiting for another PR to be merged) because I'm not sure how long we gonna take for code review in another PR.

Well, the benefit is, we can merge this pr to `master` and make a new release without needed to wait for maybe 3 - 5 days later to complete everything since those 2 prs (#23 & #24) didn't related with each other at all (we can have card brand logo settings without translation). 
So, We can ship it one by one.

For the text domain, from the reason above. I asked you because the previous code we had defined the text domain `Omise`. 
So, if we gonna ship it one by one, we should make it backward compatible as much as possible with the previous version instead of waiting for another future PR to complete the code.

(and yep, I saw that PR #24 already)